### ISSUE TITLE
inotify test fails asserting the correct number of events.

### ISSUE THREAD

**Issue by henrikbjorn** [henrikbjorn/Lurker#17]
https://travis-ci.org/henrikbjorn/Lurker/jobs/87111070

**Comment by remicollet** [comment]
Indeed, same failure when inotify extension is available (0.1.6 or 2.0.0)
Test suite ok passes when extension is not available (tests are skipped)
